pkgs/development/python-modules/arelle/default.nix

```diff
@@ -46,6 +49,8 @@ buildPythonPackage {
     (cd apidocs && make html && cp -r _build $doc)
     '';
 
+  doCheck = if gui then true else false;
+
```

Comment:
I think python2 is no longer supported. `./bin/arelle` is python3-only and https://github.com/Arelle/Arelle/blob/master/setup.py#L424 only lists python3